spec/conman_spec.rb

```diff
@@ -61,4 +86,10 @@ def expect_field(key, value)
     expect(conman.contact_of_id(0)[key]).to eq(value)
   end
 
+  def dummies
```

Comment:
Are these dummies? How are they different to the "real" contact objects?
I'd go as far and just call this method `contacts` or `contact_list`. The tests do not need to know if these are dummies, the tests just want to use a ""list of contacts".

This method could also be refactored to be a `let`'ted variable.